### PR TITLE
417 multi

### PR DESCRIPTION
**Pull request by paigerube14**

### CODE REVIEW

**Comment by paigerube14** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-multi-node-density-heavy-10nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-multi.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  as: data-path-4nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKER_VM_TYPE: m7g.metal
+      ADDITIONAL_WORKERS: "2"
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-multi.yaml:61]: @paigerube14 I think 3 worker nodes should be enough, and this is the config we have in our [config google sheet](https://docs.google.com/spreadsheets/d/1DdFiJkCMA4c35WQT2SWXbdiHeCCcAZYjnv6wNpsEhIA/edit?gid=193707991#gid=193707991) now. So let's update this to ADDITIONAL_WORKERS: "1". 4.16 job can be updated too. Tnx! 
@liqcui Liquan, let me know if you have any comments.

Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-multi.yaml:61]: I copied what was on 4.16. I'll update both 4.16 and 4.17 with 1 additional worker and 2 base workers

**Comment by paigerube14** [comment]
test failure due to build cluster issue: https://redhat-internal.slack.com/archives/CFUGK0K9R/p1724328449418199

**Comment by paigerube14** [comment]
@qiliRedHat @shahsahil264 any further comments?

**Comment by shahsahil264** [comment]
> @qiliRedHat @shahsahil264 any further comments?

Looks good to me!

**Pull request merged**